packages/framework/src/Model/Security/Roles.php

```diff
@@ -7,7 +7,6 @@
 class Roles
 {
     public const ROLE_ADMIN = 'ROLE_ADMIN';
-    public const ROLE_ADMIN_AS_CUSTOMER = 'ROLE_ADMIN_AS_CUSTOMER';
```

Comment:
Can you please tell me why you did this?

Reply:
Yes. Login as a customer not working since symfony 4.4 because in 4.4 was fixed role comparation - see this PR https://github.com/symfony/symfony/pull/31177/files. We permanently added new role for fake user account, but symfony detected role change and fake customer was immediately logound. Now, we log as customer without this new role and we don't remove session and all condition for new roles was changed to condition for this session.